### PR TITLE
Fix benchmark schema update

### PR DESCRIPTION
**Pull request by KCCPMG**
Updates initial_migrations, routers and tests for routers/iep and routers/para, and several downstream components. When this is approved and merged, remaining work on #324 can be divvied up and assigned.

### CODE REVIEW

File: src/backend/routers/iep.test.ts [hunk 1]
```diff
@@ -23,24 +23,30 @@ test("basic flow - add/get goals, subgoals, tasks", async (t) => {
...
+    status: "In Progress",
     description: "subgoal 1",
+    setup: "Setup here",
     instructions: "instructions here",
-    target_max_attempts: 5,
     materials: "materials",
-    target_level: 100,
-    baseline_level: 20,
+    target_level: 85,
+    baseline_level: 60,
```
Comment (katconnors) [inline src/backend/routers/iep.test.ts:32]: May be good to add a test for current_level as well?

Comment (katconnors) [reply src/backend/routers/iep.test.ts:32]: That makes sense! Thanks. I think a comment (or no changes) should be okay.

**Review by katconnors — APPROVED** [review]
New fields look good on the site, and the code looks good to me.

File: src/backend/routers/iep.test.ts (hunk 1 above)
Comment (KCCPMG) [reply src/backend/routers/iep.test.ts:32]: Perfect! I added a comment there and to the initial_migrations to clarify that. I kept the test I added this morning and made sure to check that the value of the returned current_level was null, to make sure it wasn't getting a value but was still being included in the returned object.

**Pull request merged**